lib/wallaby/query.ex

```diff
@@ -235,13 +236,17 @@ defmodule Wallaby.Query do
     Keyword.get(conditions, :count)
   end
 
+  def getNumber(%Query{conditions: conditions}) do
```

Comment:
Name changed.

However, I could find no tests for similar functions (count, inner_text, et al) so I followed suit.